tests/workflows/pydantic_output_reference/workflow.py

```diff
@@ -0,0 +1,50 @@
+from pydantic import BaseModel
+
+from vellum.workflows import BaseWorkflow
+from vellum.workflows.nodes.bases import BaseNode
+from vellum.workflows.outputs import BaseOutputs
+
+
+class UserProfile(BaseModel):
+    """A Pydantic model representing a user profile."""
+
+    name: str
+    age: int
+    email: str
+
+
+class FirstNode(BaseNode):
+    """A node that outputs a Pydantic model."""
+
+    class Outputs(BaseOutputs):
+        profile: UserProfile
+
+    def run(self) -> BaseOutputs:
+        return self.Outputs(
+            profile=UserProfile(
+                name="Alice",
+                age=30,
+                email="alice@example.com",
+            )
+        )
+
+
+class SecondNode(BaseNode):
+    """A node that references a field from the Pydantic model output."""
+
+    user_name = FirstNode.Outputs.profile["name"]
```

Comment:
DevinAI change this reference to point to the profile directly, eg `user = FirstNode.Outputs.profile`